sycl/doc/EnvironmentVariables.md

```diff
@@ -47,6 +47,7 @@ subject to change. Do not rely on these variables in production code.
 | `SYCL_CACHE_THRESHOLD` | Positive integer | Cache eviction threshold in days (default value is 7 for 1 week). Set to 0 for disabling time-based cache eviction. |
 | `SYCL_CACHE_MIN_DEVICE_IMAGE_SIZE` | Positive integer | Minimum size of device code image in bytes which is reasonable to cache on disk because disk access operation may take more time than do JIT compilation for it. Default value is 0 to cache all images. |
 | `SYCL_CACHE_MAX_DEVICE_IMAGE_SIZE` | Positive integer | Maximum size of device image in bytes which is cached. Too big kernels may overload disk too fast. Default value is 1 GB. |
+| `SYCL_ENABLE_DEFAULT_CONTEXTS` | '1' or '0' | Enable ('1') or disable ('0') creation of default contexts in SYCL runtime. Enabled by default on Linux and disabled on Windows. |
```

Comment:
I guess we already have "default contexts" in SYCL runtime, but they work differently. I suggest we clarify that it enables creation of "platform default context" and describe how it's behavior is different from regular "default context". Otherwise there might be a confusion. What do you think?

Reply:
Could you please clarify what "default contexts" we already have?